pom.xml

```diff
@@ -188,4 +189,11 @@
             <url>https://api.bintray.com/maven/richnorth/maven/test-containers</url>
         </repository>
     </distributionManagement>
+
+    <repositories>
+        <repository>
+            <id>jitpack.io</id>
```

Comment:
just make sure that Jitpack's dependencies will not appear during the release to Maven Central :)